### ISSUE TITLE
[5.x]: Enabling "Full Name" in Card Attributes of Users > User Profile Fields triggers "Argument must be of type array, null given"

### ISSUE THREAD

**Issue by redburn** [craftcms/cms#16358]
### Description

On a mostly clean, fresh install of Craft CMS, when I go to Settings > User Profile Fields, then under "Card Attributes" I check the box in front of "Full Name", I'm presented with the following error:

```
craft\base\Element::attributePreviewHtml(): Argument #1 ($attribute) must be of type array, null given, called in /home/dev/crafty/vendor/craftcms/cms/src/helpers/Cp.php on line 2638
```

### Steps to reproduce

1. Open Crafty admin and click Settings
2. Click Users
3. In User Profile Fields, under Card Attributes, click "Full Name"

### Expected behavior

The text "Full Name" should appear in the card on the right of the Card Attributes list.

### Actual behavior

The above error is given and the card is not updated, even when you toggle other attributes like Email or UID, until Full Name is unchecked.

### Craft CMS version

5.5.7

### PHP version

8.3.15

### Operating system and version

AlmaLinux 9.5

### Database type and version

MariaDB 10.5.22

### Installed plugins and versions

-

**Comment by brandonkelly** [comment]
Thanks for reporting that! Craft 5.5.8 is out with a fix.

**Comment by redburn** [comment]
Thanks. After the update, "Full Name" now appears twice in the list of Card Attributes, once with a value of "fullName", which is the one added by this fix, and once with a value of "layoutElement:bd39c72b-a429-4be1-82d7-d76f523d842d", which seems to be the old one that produced the error described here. Toggling that will still produce the error.

I can just ignore it so it's not really an issue, but perhaps it can be cleaned up somehow?

**Comment by brandonkelly** [comment]
Thanks for pointing that out! Fixed in 5.5.9.
